Source/FieldSolver/WarpX_QED_K.H

```diff
@@ -75,6 +73,7 @@ void warpx_hybrid_QED_push (int j, int k, int l, Array4<Real> const& Ex, Array4<
                             const Real xi_c2)
 {
 
+using namespace amrex;
```

Comment:
Sorry for the very dumb question but, If you add this line here, why do you need `amrex::` in the lines below?

Reply:
We don't, actually. We could remove them in another PR. But it doesn't hurt to keep them, and sometimes it helps us know if it's an AMReX or a WarpX element.

Reply:
Thanks @MaxThevenet! That makes sense, but then (I am just curious) why would you not do that in the function where you decided to use the namespace, instead of just referencing with `amex::`?

Reply:
do what?

Reply:
Ah, sorry, got it! we may not all have the same usage, but I would say that, for instance, `MultiFab` is so common and specific that we don't need to prefix it with `amrex::`. However, the function `grow`, for instance, could exist in many libraries, so it is worth specifying. But we don't really have a convention in WarpX, and honestly it's not consistent throughout the code.

Reply:
Ah ok, got it! That's very good to know :)